data_steward/curation_logging/slack_logging_handler.py

```diff
@@ -0,0 +1,44 @@
+# Python imports
+import os
+import logging
+
+# Project imports
+from utils.slack_alerts import post_message, check_channel_and_token, SlackConfigurationError, SLACK_CHANNEL, SLACK_TOKEN
+
+
+class SlackLoggingHandler(logging.Handler):
+    """
+     Logging handler to send messages to a Slack Channel.
+    """
+
+    def __init__(self):
+        super().__init__(level=logging.WARNING)
+
+    def emit(self, record):
+
+        try:
+            if not _is_raised_from_itself(record):
+                post_message(record.msg % record.args)
+        except SlackConfigurationError:
+            logging.exception(
+                'Slack is not configured for posting messages, refer to playbook.'
+            )
+            raise
+
+def _is_raised_from_itself(record):
+    return record.module in __name__
```

Comment:
nit: this is just a personal preference, I would define `_is_raised_from_itself` as a class method in `SlackLoggingHandler`. `self.__module__` seems more understandable than `__name__` although they refer to the same thing. 

```suggestion
    def _is_raised_from_itself(self, record):
        return record.module in self.__module__
```

Reply:
I got an error running the unit test when I defined this as a class method because I had it as `return record.module in self.__name__` NOT `return record.module in self.__module__`. I changed it and now it's working as expected!